easyaudit/admin_helpers.py

```diff
@@ -30,9 +30,14 @@ def prettify_json(json_string):
 
 
 class EasyAuditModelAdmin(admin.ModelAdmin):
+    def get_changelist_instance(self, *args, **kwargs):
+        changelist_instance = super().get_changelist_instance(*args, **kwargs)
+        user_ids = [obj.user_id for obj in changelist_instance.result_list]
```

Comment:
Minor (non-blocking), we could do `user_ids = set(...)`. It won't affect the database, but it may lower the memory usage a _little_. Non-blocking.